Task3/Task_3_Bogdan_Topalov/car_wash_db_pool/car_wash_db/INSERT_DATA.sql

```diff
@@ -0,0 +1,69 @@
+INSERT INTO "Users" (
```

Comment:
As I wrote in the chat, all insert should be splitted into files and located in the data folder. Please apply file structure changes